test/authenticator.spec.ts

```diff
@@ -495,7 +453,7 @@ describe('Decentraland Crypto', function () {
             '0xd73b0315dd39080d9b6d1a613a56732a75d68d2cef2a38f3b7be12bdab3c59830c92c6bdf394dcb47ba1aa736e0338cf9112c9eee59dbe4109b8af6a993b12d71b'
         }
       ]
-      expect(Authenticator.isValidAuthChain(chain)).to.be.equal(true)
+      expect(Authenticator.isValidAuthChain(chain)).toEqual(true)
     })
   })
 })
```

Comment:
I know that it was that way before but maybe we can move the RPC URL to a constant at the top of the file.